src/themes/components/title.ts

```diff
@@ -3,15 +3,15 @@ import { ComponentMultiStyleConfig } from '@chakra-ui/react';
 const parts = ['title', 'text'];
 
 const variants = {
-  hero: {
+  page: {
```

Comment:
I sync with the designer about the name and this is not a hero but a page title variation.